app/src/main/java/com/babylon/wallet/android/presentation/wallet/WalletScreen.kt

```diff
@@ -222,7 +223,9 @@ private fun WalletContent(
         val pullRefreshState = rememberPullRefreshState(state.isRefreshing, onRefresh = onRefresh)
         Box(modifier = Modifier.padding(padding)) {
             WalletAccountList(
-                modifier = Modifier.pullRefresh(pullRefreshState),
+                modifier = Modifier
+                    .fillMaxSize()
```

Comment:
curious why it was needed?

Reply:
If you hide the only account you have, you end up in a weird looking state, all the content being in the top left corner (the balance and the create account button)